packages/editor/src/lib/editor/Editor.ts

```diff
@@ -3224,19 +3225,6 @@ export class Editor extends EventEmitter<TLEventMap> {
 	/** @internal */
 	private readonly _renderingBounds = atom('rendering viewport', new Box())
 
-	/**
-	 * The current rendering bounds in the current page space, expanded slightly. Used for determining which shapes
-	 * to render and which to "cull".
-	 *
-	 * @public
-	 */
-	getRenderingBoundsExpanded() {
```

Comment:
Should we deprecate it?

Reply:
Probably ok to just yank, it should have been internal.